lighthouse-core/gather/gatherers/offline.js

```diff
@@ -25,7 +26,8 @@ class Offline extends Gatherer {
 
   afterPass(options, tracingData) {
     const navigationRecord = tracingData.networkRecords.filter(record => {
-      return record._url === options.url && record._fetchedViaServiceWorker;
+      return cleanHash(new URL(record._url)) === cleanHash(new URL(options.url)) &&
```

Comment:
This seems a little worrying at first (how do we account for all the crazy things SPAs do with URL fragments?), but this is in line with how service worker caching of the page works (the SW Cache explicitly ignores the URL fragment on requests), so seems like it may be the right thing to do.

@jeffposnick for his opinion